__DEFINES/setup.dm

```diff
@@ -197,6 +196,11 @@ var/MAX_EXPLOSION_RANGE = 14
 #define ALL ~0
 #define NONE 0
 
+//airflow flags!
+
+#define ON_BORDER 1   // item has priority to check when entering or leaving
+#define IMPASSIBLE 2  // item will make things auto_fail on prox checks through it
```

Comment:
Definition of impassible
1 a : incapable of suffering or of experiencing pain
b : inaccessible to injury
2 : incapable of feeling : impassive